qcelemental/tests/test_constants.py

```diff
@@ -96,3 +96,10 @@ def test_codata2022():
         qcelemental.PhysicalConstantsContext("CODATA2022")
 
     assert "only contexts {'CODATA2014', 'CODATA2018', } are currently supported" in str(e.value)
+
+
+def test_codata_comparison():
```

Comment:
It seems we should keep comparison functions in test code? It doesn't seem that keeping it on the objects in user space is the right move. I know we have had this discussion before, by my 2 cents again.

Reply:
maybe free fns in the context.py file so they don't get imported by ordinary user? if I was the only one who'd ever use them, I'd be happy for them to be in test code. but for any downstream thinking of migrating to qcel physconst, I'd like header-writing and comparison framework to be blatant and encouraging.

Reply:
I think a freestanding function would be fine. But any real difference between this free standing being in `context.py` vs `test` both would be roughly as accessible.

Reply:
absolutely, just as accessible. My worry is that the adopter won't look in tests. Or will think the fn is tied to the testing framework and not examine general applicability.